azurerm/resource_arm_traffic_manager_endpoint.go

```diff
@@ -110,6 +110,46 @@ func resourceArmTrafficManagerEndpoint() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"custom_header": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"value": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+					},
+				},
+				ForceNew: true,
+				Optional: true,
+			},
+
+			"subnet": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"first": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
```

Comment:
COuld we validate this: `validate.IPv4Address`

Reply:
Sure